code/modules/surgery/organs/eyes.dm

```diff
@@ -85,7 +85,7 @@
 /obj/item/organ/eyes/robotic
 	name = "robotic eyes"
 	icon_state = "cybernetic_eyeballs"
-	desc = "Your vision is augmented."
+	desc = "A very basic set of optical sensors with no extra vision modes or functions."
```

Comment:
the bikeshed is red

Reply:
My favorite color Bord I'm glad we finally see eye to eye!

Reply:
red sucks though

Reply:
frick...

Reply:
I honestly prefer ultraviolet